sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/jms/AbstractServiceBusJmsAutoConfigurationTest.java

```diff
@@ -87,12 +87,12 @@ void testAzureServiceBusJMSPropertiesConfigured() {
             .withPropertyValues(
                 "spring.jms.servicebus.connection-string=" + CONNECTION_STRING,
                 "spring.jms.servicebus.topic-client-id=cid",
-                "spring.jms.servicebus.idle-timeout=123")
+                "spring.jms.servicebus.idle-timeout=123s")
             .run(context -> {
                 assertThat(context).hasSingleBean(AzureServiceBusJmsProperties.class);
                 assertThat(context.getBean(AzureServiceBusJmsProperties.class).getConnectionString()).isEqualTo(CONNECTION_STRING);
                 assertThat(context.getBean(AzureServiceBusJmsProperties.class).getTopicClientId()).isEqualTo("cid");
-                assertThat(context.getBean(AzureServiceBusJmsProperties.class).getIdleTimeout()).isEqualTo(123);
+                assertThat(context.getBean(AzureServiceBusJmsProperties.class).getIdleTimeout().getSeconds()).isEqualTo(123);
             });
     }
 
```

Comment:
The test class name should be updated to align with the source code.
~~AbstractServiceBusJmsAutoConfiguration.java~~ -> AbstractServiceBusJmsAutoConfigurationTests.java
ServiceBusJmsAutoConfiguration.java -> ServiceBusJmsAutoConfigurationTests.java

Reply:
new UT added already now